libbpf-tools/filelife.h

```diff
@@ -0,0 +1,15 @@
+/* SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause) */
+#ifndef __FILELIFE_H
+#define __FILELIFE_H
+
+#define DNAME_INLINE_LEN 32
+#define TASK_COMM_LEN    16
+
+struct event {
+	char file[DNAME_INLINE_LEN];
+	char task[TASK_COMM_LEN];
+	__u64 delta_ns;
+	pid_t tgid;
```

Comment:
move these two up? char arrays have 1-byte alignment so best put at the end; fields should generally be sorted by alignment size, not field size

Reply:
Oh, got it. I'll fix this. Thank you for point this!

Reply:
@anakryiko Fixed, I realized the tools I submitted previously had this problem too. I'll fix them in another PR with `get_ktime_ns` to trace_helpers.c then keep converting others another day. Thanks for your continued guidance!